### DISCUSSION TITLE
Welcome to The Boring JavaScript stack Discussions!

### DISCUSSION THREAD

**Discussion by DominusKelvin** [sailscastshq/boring-stack#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by d-a-ve** [comment]
Totally didn't read this until I brought up an issue 😅.

I'm Dave, I am a Nigerian based frontend developer with a boring tech stack of React and typescript 🥱. 

I look forward to contributing to the project and meeting the rest of the community.
